src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -333,14 +347,31 @@ public void Dispose()
             HomeWorkspaceModel.WorkspaceClosed -= this.CloseExtensionTab;
             PackageDependencyTable.ItemsSource = null;
             LocalDefinitionsTable.ItemsSource = null;
+            ExternalFilesTable.ItemsSource = null;
             dataRows = null;
             localDefinitionDataRows = null;
+            externalFilesDataRows = null;
         }
 
         private void Refresh_MouseLeftButtonDown(object sender, System.Windows.Input.MouseButtonEventArgs e)
         {
             DependencyRegen(currentWorkspace);
         }
+
+        /// <summary>
+        /// Computes the file size from the path.
+        /// </summary>
+        private string GetFileSize(string path)
```

Comment:
This can probably be moved to `DynamoUtilities` project -> File Path helper

Reply:
Yes, will move this in the next commit.